addons/project/views/project_sharing_project_task_views.xml

```diff
@@ -151,12 +151,8 @@
                                 class="oe_stat_button" icon="fa-repeat" context="{'default_user_ids': [(6, 0, [uid])]}">
                             <field name="recurring_count" widget="statinfo" string="Recurring Tasks"/>
                         </button>
-                        <button name="action_project_sharing_open_subtasks" type="object" class="oe_stat_button" icon="fa-tasks"
-                            invisible="not id or subtask_count == 0" context="{
-                                'default_user_ids': [(6, 0, [uid])],
-                                'default_project_id': project_id,
-                                'default_display_in_project': False,
```

Comment:
is it expected to remove that in the context? 🧐 

I don't see the point in the spec saying to remove that. 🤔

Reply:
It seems the context has be added later on and might have been added due to rebase, putting them back in. :disappointed: